### PR TITLE
update rprofile

### PR DESCRIPTION
**Pull request by dgrassellyb**

### CODE REVIEW

File: .Rprofile [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
     Sys.setenv("RENV_PROFILE" = renv_profile)
   } else {
-    message("This repository do not contains the renv profile for your R version.")
+  message("This repository do not contains the renv profile for your R version.")
   }
-  source("renv/activate.R")
-} else {
+}
+
+if ((Sys.getenv("GITHUB_ACTIONS") != "") || (Sys.getenv("DOCKER_CONTAINER_CONTEXT") != "")) {
```
Comment (galachad) [inline .Rprofile:13]:
```suggestion
if (Sys.getenv("GITHUB_ACTIONS") != "") {
```

Remove `DOCKER_CONTAINER_CONTEXT` condition.

File: .Rprofile [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
   options(repos = c(CRAN = "https://cran.rstudio.com"))
+  Sys.setenv(RENV_AUTOLOADER_ENABLED=FALSE)
 }
+source("renv/activate.R")
```
Comment (galachad) [inline .Rprofile:17]:
```suggestion
source("renv/activate.R")

```

Add EOF.

**Review by galachad — APPROVED** [review]
Approve, please take a look on suggestions.

**Comment by galachad** [comment]
@dgrassellyb for situation where we do not want to load the renv, we can set the `RENV_AUTOLOADER_ENABLED` in shell script.

E.g. 

```
export RENV_AUTOLOADER_ENABLED="false"
```

Useful when do not want to load renv.

**Comment by dgrassellyb** [comment]
> @dgrassellyb for situation where we do not want to load the renv, we can set the `RENV_AUTOLOADER_ENABLED` in shell script.
> 
> E.g.
> 
> ```
> export RENV_AUTOLOADER_ENABLED="false"
> ```
> 
> Useful when do not want to load renv.

yeah will probably need to do that only for github actions

File: .devcontainer/codespaces_rstudio_status.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    exit 0
+  else
+    echo "$http_status"
+    sleep $WAIT_TIME
+    ((retries++))
+  fi
+done
+
+echo "Error accessing rstudio ($url_rstudio) - Timeout reached."
+exit 1
```
Comment (galachad) [inline .devcontainer/codespaces_rstudio_status.sh:28]: Looks like it was a problem with url that not redirect correctly (localhost error). We can add the shell script call `rstudio` to bin, but please let's do in separate pr. 

Example:
`"https://$url-8787.preview.app.github.dev"` should be changed to `"https://$url-8787.app.github.dev"`
